docs/content/comparisons/Druid-vs-commericial-solutions.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: doc_page
+---
+
+Druid vs Proprietary Commercial Solutions (Vertica/Redshift)
+============================================================
+
+The propietary database world has numerous solutions by numerous vendors. At a very high level, Druid is distinguished from most  
+of these solutions by the scale at which it can operate at, the scale and performance seen with ingesting streaming data, the performance seen from issuing ad-hoc exploratory queries, 
+the support it has for multi-tenancy (making Druid ideal for powering user facing applications), and how cost effective it is to run at scale. Druid is an open source project and the development 
+is community led. The direction of the project is driven by the needs of the community. Below, we highlight comparisons between some specific 
+systems. We welcome contributions for additional comparisons.
+
+## Druid vs Redshift
+
+### How does Druid compare to Redshift?
+
+In terms of drawing a differentiation, Redshift is essentially ParAccel (Actian) which Amazon is licensing.
```

Comment:
Amazon reps have stated they have changed the source significantly, rather than just a drop-in. I don't have any verification on how true that is or how "significant" their changes are.